doc/source/references.rst

```diff
@@ -1,14 +1,6 @@
 References
 ==========
 
-.. [AndBorChaDelGiacLewWhe] J. E. Andersen, G. Borot, S. Charbonnier,
```

Comment:
These references are not used anywhere. To make sphinx happy, we should use them somewhere or remove them.

Reply:
Actually, the algorithm is used, let me add a ref.

Reply:
The reference is already there in `surface_dynamics.topological_recursion.masur_veech`. I guess this module is nowhere referenced in the sphinx doc.

Reply:
I am making a commit.

Reply:
Fixed in af8214c